### DISCUSSION TITLE
JSON error when loading app on hosted droplet

### DISCUSSION THREAD

**Discussion by jakeusu** [Budibase/budibase#1884] [Help]
When I access budibase using the IP address:10000...it works fine.

But when I click on the app I'm building, it does not load and displays this error: 

Something went wrong: Unexpected token u in JSON at position 0

**Comment by Rory-Powell** [comment]
Hi @jakeusu. This sounds like it could be a session related issue, could you try clearing your browser cookies for budibase?
I've seen this occur when multiple installations are accessed on the same browser sesson, did you by chance run it locally before switching to the droplet?

**Reply by Rory-Powell** [reply]
Alternatively you could try a different browser or a private / incognito session

**Reply by jakeusu** [reply]
No local before switching. I'll clean the browser cookies and try again.
